lib/Yam/agama/agama_base.pm

```diff
@@ -33,6 +37,10 @@ sub post_run_hook {
     ensure_serialdev_permissions;
 }
 
+sub set_password {
```

Comment:
there is not need for additional method here to set or export it afik, just use it in the right place